src/main/java/uk/gov/companieshouse/limitedpartnershipsapi/controller/PartnershipController.java

```diff
@@ -162,12 +163,7 @@ public ResponseEntity<List<Cost>> getCosts(
             @PathVariable(URL_PARAM_SUBMISSION_ID) String submissionId,
             @RequestHeader(value = ERIC_REQUEST_ID_KEY) String requestId) {
 
-        var logMap = new HashMap<String, Object>();
-        logMap.put(TRANSACTION_KEY, transaction.getId());
-        ApiLogger.infoContext(requestId, "Calling CostsService to retrieve costs", logMap);
-
-        Cost cost = costsService.getTemporaryZeroCost(submissionId, "Limited Partnership", requestId);
-
-        return ResponseEntity.ok(Collections.singletonList(cost));
+        // TODO Implement cost for 'change of Limited Partnership name' post transition journey.
+        return ResponseEntity.status(HttpStatus.NOT_IMPLEMENTED).body(null);
```

Comment:
Keeping this costs endpoint as we'll need it for the change of name functionality.

Reply:
Actually. Will remove as it's incomplete functionality.